README.rst

```diff
@@ -166,9 +166,6 @@ Volume Control
 
     $ songpal volume 20
 
-    $ songpal volume +5
-
-    $ songpal volume -10
```

Comment:
This should not be removed. It does not work on all devices (known buggy firmwares), but it can still be useful for someone. If UPnP part does not support this, there are two potential solutions: either fetch the current volume and adjust it accordingly, or simply error out saying it is not supported.

Reply:
Then then code must be adjusted accordingly.

The type hint currently tells that set volume method takes an int, and the code then coverts it into str. So the documentation does not match the implementation. I did change that first, but then after found out that consumer of this library (home assistant songpal integration) is calling it with an int, I decided to switch back to the original signature (using int) and update the documentation to reflect what the code supports

Reply:
Had an implementation as you suggested (https://github.com/rytilahti/python-songpal/pull/84/commits/7b5114fa298994c75cf74828e3e63a7cc9ed5310#diff-876d560a154744deca94b669fc4fdf5dR339)

Reply:
Yeah, the type hint is indeed incorrectly set, it should be `Union[str, int]` (to allow both relative and absolute values).